wiki/Tournaments/SS_Tournament/en.md

```diff
@@ -0,0 +1,91 @@
+# SS Tournament 2018
+
+![SST Tournament 2018](Banner.png)
+
+SS Tournament 2018 is a osu! standard tournament organized and ran by by ![][flag_CA] [Sparhten](https://osu.ppy.sh/users/7601720), [NeonLights](https://osu.ppy.sh/users/5277199), and ![] [flag_GE] (https://osu.ppy.sh/users/9696149).
```

Comment:
you use wrong Markdown Img Syntax, and you forgot to put flag refrence to this article

Reply:
how do i fix this? im still decently new to git hub